demos/notebooks/demo_dendritic.ipynb

```diff
@@ -330,16 +384,16 @@
     "grid_shape = (np.ceil(np.sqrt(Nc/2)).astype(int), np.ceil(np.sqrt(Nc*2)).astype(int))\n",
     "plt.figure(figsize=np.array(grid_shape[::-1])*1.5)\n",
     "plt.imshow(montage(A, rescale_intensity=True, grid_shape=grid_shape))\n",
-    "plt.title('Montage of found spatial components');\n",
+    "plt.title(f'Montage of initial spatial components alpha={alpha}');\n",
```

Comment:
There's a newer syntax for this, but I don't know if our minimum python version lets us use it yet.

Reply:
Ah, yes. Looks like the new syntax is 3.8 and up (our min version is 3.8). It's up to you if you like and want to use it or not.

In Python 3.8, you can replace

f"foo={foo}"

with
f"{foo=}"